src/easy/handler.rs

```diff
@@ -378,6 +379,8 @@ pub fn ssl_ctx(cx: *mut c_void) -> Result<(), Error> {
 /// ```
 pub struct Easy2<H> {
     inner: Box<Inner<H>>,
+    /// Mime handles to free upon drop
+    mimes: Vec<*mut curl_sys::curl_mime>,
```

Comment:
Only one mime object should be attached to an easy handle at a time since `CURLOPT_MIMEPOST` replaces any previously assigned `mime`, so this doesn't seem like it needs to be a `Vec`.

This should also probably go inside `Inner` like everything else.

I would also prefer to avoid a raw pointer here since that requires multiple implementations handling the calling of `curl_mime_free` in multiple places. I wonder if there is a way of keeping the drop code in the `Mime` struct?

Reply:
Ok. Done.